Doc/using/configure.rst

```diff
@@ -1495,6 +1495,15 @@ Linker flags
 
    .. versionadded:: 3.8
 
+.. envvar:: PY_BUILTIN_MODULE_LDFLAHS
+
+   Linker flags to build a standard library extension module as a built-in
+   module, like the :mod:`posix` module.
```

Comment:
Perhaps make it clearer that this applies to all builtin modules.

Reply:
Done